index.js

```diff
@@ -89,6 +111,8 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+var myArrowFunction = () => {return 'I called an arrow function!'};
+console.log(myArrowFunction());
```

Comment:
nice!
This will definitely work, but if you are using curly braces for a code block, you should start a newline when you open them, so this could be
```
var myArrowFunction = () => {
  return 'I called an arrow function!'
};
```
If you want to write it on one line to be more concise, yo ucan do that like this..
`const myArrowFunction = () => "This is a function expression!";`
^ You're allowed to write it like that when there's nothing else you need to do in the code block beside return. It's sort of a shorthand. so you just take out the curly brackets and the return keyword, then javascript understands directly after the arrow is what you want to return.